ansible/roles/monitoring-influxdb9/tasks/main.yml

```diff
@@ -117,3 +121,23 @@
     - rm /tmp/influxdb.users.sh
   tags:
     - influx_users
+
+# Copy Influx backup script to influx hosts
+- name: Influx backup script to influx hosts
+  template: src=influxdb.backup.sh.j2  dest=/tmp/influxdb.backup.sh mode=0775
+  tags:
+    - influx_backup
+
+# Running backup script as a cronjob on influx hosts
```

Comment:
There is a cron module for ansible. E.g. in roger-mesos/ansible//roles/zookeeper/tasks/main.yml